analysis_options.yaml

```diff
@@ -21,7 +21,7 @@ analyzer:
   ]
   strong-mode:
     implicit-casts: false
-    implicit-dynamic: false
+    # implicit-dynamic: false # at least temporary for RecursiveVisitor change
```

Comment:
Is there any way to just opt-out the line or file specifically?

We should also file a TODO to uncomment this when the SDK rolls.

Reply:
TODO added. I can't make opting out only the line work :(